src/screens/SignTransactionSheet.tsx

```diff
@@ -2108,7 +2112,8 @@ const SimulatedEventRow = ({
   if (asset?.type === TransactionAssetType.Native) {
     assetCode = ETH_ADDRESS;
   }
-
+  const showUSD = eventType === 'send';
```

Comment:
is there a reason to only show this for send? if we have this data we should be showing it receives as well